.eslintrc.js

```diff
@@ -54,14 +54,15 @@ module.exports = {
 
     'import/no-useless-path-segments': 'error',
     'import/order': [
-      'error',
+      'warn',
```

Comment:
please add TODO comment for this, i suggest to comment the entire import/order eslint for now

Reply:
I can't comment it because of airbnb eslint config, but I can turn it off and add todo on it. 
TODO added and the import order turned off temporarily.